test/model/message_test.dart

```diff
@@ -576,4 +603,197 @@ void main() {
         .reactions.isNotNull().jsonEquals([eg.unicodeEmojiReaction]);
     });
   });
+
+  group('handleSubmessageEvent', () {
+    late StreamMessage streamMessage;
+
+    Future<int> preparePollMessage({List<PollOption>? options}) async {
+      List<PollOption> effectiveOptions = options
+        ?? [PollOption(text: 'foo'), PollOption(text: 'bar')];
+      const messageId = 123;
+      streamMessage = eg.streamMessage(
+        id: messageId,
+        sender: eg.selfUser,
+      );
```

Comment:
nit: fold onto one line

In general, since I'm seeing several cases of this in this PR: using four lines for something that can naturally fit on one is expensive in terms of how much can fit on the screen, and therefore of how far one can look around at surrounding code without scrolling back and forth. So it's good to keep things on fewer lines when they fit easily without cramping.